src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -264,6 +264,49 @@ def recover_keyvault(cmd, client, vault_name, resource_group_name, location):
                                    parameters=params)
 
 
+def _parse_network_acls(cmd, resource_group_name, network_acls):
+    if not network_acls:
+        return None
+
+    logger.info('network_acls: {}'.format(network_acls))
+
+    try:
+        network_acls_json = json.loads(network_acls)
+    except:  # pylint: disable=bare-except
+        with open(network_acls) as f:
+            logger.info('network_acls_filename: {}'.format(network_acls))
+            network_acls_json = json.load(f)
+
+    try:
+        network_acls_json = json.loads(json.dumps(network_acls_json))
```

Comment:
why load again?

Reply:
> why load again?

@yungezz Good point, actually I added this sentence in order to make sure the input is a valid JSON. But later I modified the code that makes this part useless... I will remove this part.